schema/mvp.json

```
@@ -1181,13 +1181,13 @@
                 {
                   "name": "2",
                   "src": "/images/mobile.png",
-                  "altText": "phone - calling your friend to ask for help",
+                  "altText": "Calling your friend to ask for help",
```

I think this makes the tense more consistent with the other option (`Head to the library`)?
```suggestion
                  "altText": "Call your friend to ask for help",
```